hibernate-c3p0/src/main/java/org/hibernate/c3p0/internal/C3P0ConnectionProvider.java

```diff
@@ -121,31 +125,32 @@ public void configure(Map<String, Object> props) {
 				JdbcSettings.URL,
 				JdbcSettings.JPA_JDBC_URL
 		);
+
 		final Properties connectionProps = ConnectionProviderInitiator.getConnectionProperties( props );
 
-		C3P0_MSG_LOGGER.c3p0UsingDriver( jdbcDriverClass, jdbcUrl );
-		C3P0_MSG_LOGGER.connectionProperties( ConfigurationHelper.maskOut( connectionProps, "password" ) );
+		ConnectionInfoLogger.INSTANCE.connectionProperties( ConfigurationHelper.maskOut( connectionProps, "password" ) );
```

Comment:
If we log this for `C3P0ConnectionProvider`, we should also log it for Agroal, Hikari, Proxool, etc.

(But I'm not sure we need to.)

Reply:
> (But I'm not sure we need to.)

That's not a very definitive answer is it though? ;)
I'll remove it, but something inside me tells me that we might be putting that back in the not so distant future ...